planet/clients/orders.py

```diff
@@ -401,23 +399,19 @@ async def poll(
 
             completed = state in states
             if not completed:
-                sleep_time = max(wait-(time.time()-t), 0)
+                sleep_time = max(wait - (time.time() - t), 0)
                 LOGGER.debug(f'sleeping {sleep_time}s')
                 await asyncio.sleep(sleep_time)
         return state
 
-    async def list_orders(
-        self,
-        state: str = None,
-        limit: int = None,
-        as_json: bool = False
-    ) -> typing.Union[typing.List[Order], dict]:
+    async def list_orders(self,
+                          state: str = None,
+                          limit: int = None) -> typing.List[Order]:
         """Get all order requests.
 
         Parameters:
             state: Filter orders to given state.
             limit: Limit orders to given limit.
-            as_json: Return orders as a json dict.
```

Comment:
This method will be much easier to maintain if it only returns a list of Orders.